### ISSUE TITLE
Create standard Github templates for Issues (adding content)

### ISSUE THREAD

**Issue by wslyvh** [wslyvh/useWeb3#29]
Since all content has moved to the repo itself, it's a little bit more work to keep tags/languages up to date.
A Github Issue template could help standardize

**Comment by RishiKumarRay** [comment]
@wslyvh do you want it in yaml format?

**Comment by wslyvh** [comment]
Hi @RishiKumarRay, that's for picking this up!
If I'm correct, it can be in markdown or yaml? Either would be fine.

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates
I think a PR template would make most sense.

Some information and guidelines that should be included in the template:

1. Resources should be submitted in markdown format in their respective sub folders in `/content` 

2. The following properties can be used
```
  title: string
  description: string
  content: string
  authors: Array<string>
  date?: Date
  level: 'All' | 'Beginner' | 'Intermediate' | 'Advanced'
  tags: Array<string>
  languages: Array<string>
  url: string
  alternateUrl?: string
  dateAdded: Date 
```
3. The filename should be a 'slugified' version of the title

4. Date formats should be in: YYYY-MM-DD 

5. Properties with a `?` are optional 

6. The following tags are allowed. Don't use any other tags. If you feel like something is missing, please create a separate issue/PR
`Smart Contracts, Dapp, Web3, DeFi, Security, EVM, DevEx, Cryptography, UX & Design, Tokens, DAO, Storage, IPFS, NFT, IDE, Privacy, Architecture, Scalability, Mobile`

6. The following languages are currently used. Feel free to add new languages, but only submit actual programming languages (not protocols or other tags)
Solidity, Rust, Cairo, JavaScript, Vyper, Python



Thank you! 🙏

**Comment by thirtyninetythree** [comment]
I think it should be added on the content/tutorials page so it's easier to find

**Comment by wslyvh** [comment]
Closed with #102
